cat.json

```diff
@@ -0,0 +1,264 @@
+              "volume": "20oz",
+              "price": 19.99
+            },
+
+            {
+              "name": "Kittin",
+              "volume": "20oz",
+              "price": 19.99
+            },
+
+            {
+              "name": "Kittin",
+              "volume": "20oz",
+              "price": 19.99
+            },
+
+            {
+              "name": "Kittin",
+              "volume": "20oz",
+              "price": 19.99
+            }
+          ]
+        }
+      ]
+    },
+
+    {
+      "name": "Meow Meal",
+      "types": [
+        {
+          "type": "Manx",
+          "volumes": [
+            {
+              "name": "Kittin",
+              "volume": "24oz",
+              "price": 19.99
+            },
+            {
+              "name": "Kittin",
+              "volume": "48oz",
+              "price": 34.99
+            },
+
+            {
+              "name": "Adult",
+              "volume": "28oz",
+              "price": 22.99
+            },
+
+            {
+              "name": "Adult",
+              "volume": "56oz",
+              "price": 40.99
+            }
+          ]
+        },
+        {
+          "type": "Egyption Mau",
+          "volumes": [
+
+            {
+              "name": "Kittin",
+              "volume": "24oz",
+              "price": 19.99
+            },
+            {
+              "name": "Kittin",
+              "volume": "48oz",
+              "price": 34.99
+            },
+
+            {
+              "name": "Adult",
+              "volume": "28oz",
+              "price": 22.99
+            },
+
+            {
+              "name": "Adult",
+              "volume": "56oz",
+              "price": 40.99
+            }
+          ]
+        },
+
+        {
+          "type": "Himalayan",
+          "volumes": [
+
+            {
+              "name": "Kittin",
+              "volume": "24oz",
+              "price": 19.99
+            },
+            {
+              "name": "Kittin",
+              "volume": "48oz",
+              "price": 34.99
+            },
+
+            {
+              "name": "Adult",
+              "volume": "28oz",
+              "price": 22.99
+            },
+
+            {
+              "name": "Adult",
+              "volume": "56oz",
+              "price": 40.99
+            }
+          ]
+        },
+        {
+          "type": "Rag Doll",
+          "volumes": [
+
+            {
+              "name": "Kittin",
+              "volume": "24oz",
+              "price": 19.99
+            },
+            {
+              "name": "Kittin",
+              "volume": "48oz",
+              "price": 34.99
+            },
+
+            {
+              "name": "Adult",
+              "volume": "28oz",
+              "price": 22.99
+            },
+
+            {
+              "name": "Adult",
+              "volume": "56oz",
+              "price": 40.99
+            }
+          ]
+        }
+      ]
+    }
+  ]
+}
```

Comment:
I _think_ your data may not match the data given, the first set of data just has the "Kittin" size/price over and over.

Also, your choice of data representation for the data given is not ideal, there is a lot of repetition going on.  A better solution would be to add a new key to each brand, whose value is an array of all the breeds it applies to, since the data for the actual food types/sizes/prices doesn't differ between the breeds.